test/run_pass/test_os.js

```diff
@@ -35,3 +35,21 @@ var expected = [
 ];
 
 assert.deepStrictEqual(actual, expected);
+
+// test for tmpdir
```

Comment:
A default case shall be covered. i.e. no `TMPDIR` nor `TMP` and `TEMP` environs are set.

Reply:
Ok, BTW I just copied these tests from here: https://github.com/nodejs/node/blob/master/test/parallel/test-os.js#L46-L78.